src/main/java/filemanager/service/impl/FeedExporter.java

```diff
@@ -53,21 +61,31 @@ public void startExport(Command command) {
                     job.setTargetTime(job.getTargetTime());
             }
             job.setStatus(JobStatus.COMPLETE);
-            counters.get(Fields.COMPLETED_TRANSACTION_COUNTER).inc();
+            TaggedMetricName jobMetricName = new TaggedMetricName(FeedExporter.class, Fields.COMPLETED_TRANSACTION_COUNTER, tags);
+            counters.put(jobMetricName.getName(), metricRegistry.counter(jobMetricName.getName()));
```

Comment:
I think you may use ```putIfAbsent``` method here, and in that way avoid re-creating metric on each execution. Otherwise I don't see a reason for you to first put that metric to the Map and then immediately get it back to increase.